code/modules/power/singularity/emitter.dm

```diff
@@ -266,3 +266,16 @@
 			user << "<span class='danger'>\The [src] is damaged.</span>"
 		if(77 to 99)
 			user << "<span class='warning'>\The [src] is slightly damaged.</span>"
+
+//R-UST port
+/obj/machinery/power/emitter/proc/get_initial_fire_delay()
+	return 100
+
+/obj/machinery/power/emitter/proc/get_rand_burst_delay()
+	return rand(min_burst_delay, max_burst_delay)
+
+/obj/machinery/power/emitter/proc/get_burst_delay()
+	return 2

```

Comment:
Is it possible to make this a var on the item, rather than a hardcoded number in a proc? Not super important, but it might come up.